installer/tap-windows6.nsi

```diff
@@ -38,9 +38,9 @@ InstallDirRegKey HKLM "SOFTWARE\${PRODUCT_NAME}" ""
 ;--------------------------------
 ;Modern UI Configuration
 
-Name "${PRODUCT_NAME} ${PRODUCT_VERSION}"
+Name "${PRODUCT_NAME} ${PRODUCT_VERSION}-${PRODUCT_TAP_WIN_BUILD}"
 
-!define MUI_WELCOMEPAGE_TEXT "This wizard will guide you through the installation of ${PRODUCT_NAME}, a kernel driver to provide virtual tap device functionality on Windows originally written by James Yonan.\r\n\r\nNote that the Windows version of ${PRODUCT_NAME} will only run on Windows XP or later.\r\n\r\n\r\n"
+!define MUI_WELCOMEPAGE_TEXT "This wizard will guide you through the installation of ${PRODUCT_NAME}, a kernel driver to provide virtual tap device functionality on Windows originally written by James Yonan.\r\n\r\nNote that ${PRODUCT_NAME} will only run on Windows Vista or later.\r\n\r\n\r\n"
```

Comment:
This change does not really belong to this commit, so if you want, I can make it separate. In any case the original wording was confusing as it was copy-and-paste from the OpenVPN installer.